algorithms/matrix/matrix chain recurssion.py

```diff
@@ -0,0 +1,17 @@
+from sys import maxint
+import time
+def matrix(p,i,j):
+    if i==j:
+        return 0
+    min=maxint
```

Comment:
do not use overlapping variable name with Python native functions. maybe rename to `minimum`?